toolbox/library/include/WBToolbox/Block/RelativeJacobian.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef WBT_RELATIVEJACOBIAN_H
+#define WBT_RELATIVEJACOBIAN_H
+
+#include "WBToolbox/Base/WBBlock.h"
+
+#include <memory>
+#include <string>
+
+namespace wbt {
+    namespace block {
+        class RelativeJacobian;
+    } // namespace block
+} // namespace wbt
+
+namespace blockfactory {
+    namespace core {
+        class BlockInformation;
+    } // namespace core
+} // namespace blockfactory
+
+/**
+ * @brief The wbt::RelativeJacobian class
+ *
+ * @section Parameters
+ *
+ * In addition to @ref wbblock_parameters, wbt::RelativeJacobian requires:
+ *
+ * | Type | Index | Rows  | Cols  | Name  |
+ * | ---- | :---: | :---: | :---: | ----- |
+ * | ::STRING | 0 + WBBlock::NumberOfParameters | 1 | 1 | "Frame" |
```

Comment:
This does not seem to be consistent with https://github.com/robotology/wb-toolbox/pull/226/files#diff-2b9b96a62b999213b36e92d8d6259e05232da379e1c2a6e722f67ee1f35f8b5cR81 .

Reply:
Updated in https://github.com/robotology/wb-toolbox/pull/226/commits/c70ea7d4a826fc7d2fdd37aaf7a1634cf7033ec2